### PR TITLE
librevenge: Update to 0.0.5

### PR DESCRIPTION
**Pull request by A108384**
**Summary**
- Update from 0.0.4 to [0.0.5](https://sourceforge.net/projects/libwpd/files/librevenge/librevenge-0.0.5)
- Add homepage to package.yml

**Test Plan**
- [x] Successfully Build and Install new Package
- [X] Successfully import a Word Perfect document into Libreoffice

**Checklist**
- [X] Package was built and tested against unstable

### CODE REVIEW

File: packages/l/librevenge/package.yml [hunk 1]
```diff
@@ -14,7 +15,7 @@ builddeps  :
     - pkgconfig(cppunit)
     - libboost-devel
 setup      : |
-    %configure --disable-werror
+    %reconfigure --disable-werror
```
Comment (EbonJaeger) [inline packages/l/librevenge/package.yml:18]: What was the reason for this change?

**Comment by A108384** [comment]
Building with %configure resulted in this error:

configure: error: unrecognized option: '--runstatedir=/run'


So I re-tried with %reconfigure which allowed the build to complete
successfully.

**Comment by EbonJaeger** [comment]
We have a macro for this case, so future packagers can easily see what's going on:

`%configure_no_runstatedir`:
> Runs `%configure` without the `--runstatedir` option. Use if you encounter `configure: error: unrecognized option: '--runstatedir=/run'`.

**Comment by A108384** [comment]
Yes, I understand.  And I wasn't really sure whether to use %reconfigure or %configure_no_runstatedir.

But the [instructions](https://help.getsol.us/docs/packaging/troubleshooting-packaging#an-old-package-using-configure-doesnt-build) seemed to indicate to try %reconfigure first.  It basically says to replace %configure with %reconfigure and try again.   Then it says:  If **_%reconfigure_** produces this error to use the new %configure_no_runstatedir macro.

At least that is how I interpreted it.

Would you like to me to change it and resubmit?

**Comment by EbonJaeger** [comment]
Ah, I forgot about that. Nevermind!

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**